### PR TITLE
feat(ui): update styles to better reflect new branding

### PR DESCRIPTION
**Pull request by johngruen**
Ready to review...

Here is a [link](https://drive.google.com/drive/folders/1QgDyzTzUAtOV8TuYkiDLMZ3sUNJ-hNwJ) to screenshots of the update (h/t @vladikoff )

This PR changes font weights, spacing, button and input sizes in order to more closely align FxA with upcoming brand guidelines. It also updates some layout CSS to use Flexbox, removes the Firefox Icon as a standalone element (it's now inside a ::before pseudo-selector), and adds a footer element into HTML templates in lieu of just having Mozilla logo to make positioning content on the page less 'squishy'.

### CODE REVIEW

**Comment by philbooth** [comment]
@johngruen, could you also tweak the commit message so that it plays nicely with our new release script? Just prefixing something `feat(ui): ` should do it, thanks.

**Comment by johngruen** [comment]
@philbooth I noticed last time my tests failed on something to do with the QR code. I'm changing styles around, but I don't think anything i'm doing affects the QR code. Is that a failure I should be worried about or something that just sometimes _happens_?

**Comment by philbooth** [comment]
> I noticed last time my tests failed on something to do with the QR code. I'm changing styles around, but I don't think anything i'm doing affects the QR code. Is that a failure I should be worried about or something that just sometimes *happens*?

With the caveat that I don't really know what I'm talking about, afaik it's not something we expect to be an intermittent failure. And most of the things that are known intermittent failures tend to succeed more often than they fail.

However, I'm also aware that some of our tests around styling are quite brittle, as they're tightly coupled to expected rules/values. So it's possible to break tests for good reasons and then have to fix them up to work with new stuff.

I'm about to nip out for a couple of hours right now, but I can take a closer look at your specific failures when I get back, if you're still having issues.

File: packages/fxa-content-server/app/styles/_breakpoints.scss [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
   trustedUI: '(max-width: 319px)',
   reasonableUI: '(min-width: 320px)',
-  small: '(max-width: 520px), (orientation:landscape) and (min-width:481px) and (max-height: 480px)',
-  big: '(min-width: 521px) and (min-height: 481px)',
+  small: '(max-width: 520px), (max-height: 640px), (orientation: landscape) and (min-width: 641px) and (max-height: 640px)',
+  big: '(min-width: 521px) and (min-height: 641px)',
   balloonSmall: '(max-width: 927px)',
   balloonBig: '(min-width: 927px)',
-  compressedLandscape: '(orientation:landscape) and (min-width:481px) and (max-height: 480px)'
+  //compressedLandscape: '(orientation:landscape) and (min-width:481px) and (max-height: 480px)'
```
Comment (vbudhram) [inline packages/fxa-content-server/app/styles/_breakpoints.scss]: Should we remove this line?

File: packages/fxa-content-server/app/styles/_base.scss [hunk 2]
```diff
@@ -144,9 +157,10 @@ input[type='radio'] {
 
 .prefillEmail {
   color: $grey-60;
-  font-size: 15px;
+  font-size: 20px;
```
Comment (vbudhram) [inline packages/fxa-content-server/app/styles/_base.scss]: CSS variable this?

File: packages/fxa-content-server/app/styles/modules/_avatar.scss [hunk 3]
```diff
@@ -144,19 +134,19 @@
...
-        width: 50px;
+        height: 46px;
+        width: 46px;
       }
     }
   }
 
   img {
-    background-color: $content-background-color;
+    background-color: #c3cfd8;
```
Comment (vbudhram) [inline packages/fxa-content-server/app/styles/modules/_avatar.scss]: Should we make this a variable?

**Comment by johngruen** [comment]
thanks @vbudhram ... i will talk to vlad about running his screenshots script in an rtl lang

**Comment by johngruen** [comment]
Fixes #892

**Comment by shane-tomlinson** [comment]
@johngruen - we have a new policy on this repo to [GPG sign all commits](https://help.github.com/en/articles/managing-commit-signature-verification) to help ensure commits actually come from the claimed authors. Mind adding your signature to this?

**Comment by johngruen** [comment]
Okay @shane-tomlinson added my GPG key

File: packages/fxa-content-server/app/styles/_layout.scss [hunk 4]
```diff
@@ -90,21 +70,40 @@
...
+  width: 100%;
+  height: 96px;
+  display: flex;
+  align-items: center;
+  padding: 0 32px;
+}
 
-  display: block;
+#stage {
+  width: 100%;
```
Comment (shane-tomlinson) [inline packages/fxa-content-server/app/styles/_layout.scss]: Can you sort properties alphabetically?

**Comment by shane-tomlinson** [comment]
@johngruen - I gave this a pretty reasonable test, not exhaustive, but most things I could think of. Overall it looks good, here are some edges that could use smoothed out:

### In the settings page on mobile, the Firefox logo and avatar are slightly off the screen.

icons off left of screen

### In settings, if the email or display name is wide, on mobile it's impossible to see the left side of the screen

mobile-can't-see-left-side-of-screen-with-long-email-display-name
ios-can't-see-left-side-long-email-display-name

### Similarly, the tos/pp links are too far to the right on mobile if email/display name is long.

Screenshot 2019-04-24 at 07 33 12


### If the email/display name is short, the mobile links aren't centered on mobile anyways

tos-pp-not-centered-mobile

### On mobile, the button text is very heavy compared to the heading

botton-text-too-heavy-compared-to-title-on-mobile

### If you look closely, the dotted border does not align with the blue outline

dotted border out of alignment

### In Choose What To Sync, the labels are a hair too close to the checkbox

labels-too-close-to-checkbox

### In the TOS/PP agreement, headers are on the wrong side in RTL

tos-headers-wrong-side-rtl

### When entering a password for recovery key, the "show password" label is visible in RTL

show-password-text-rtl-recovery-key-enable

### Finally, no avatar is visible if an avatar is set on the account, the user is signed in, and the email-first flow is opened (http://127.0.0.1:3030?action=email)

Screenshot 2019-04-24 at 07 16 03
no-image-ios-after-signed-in-action=email



Finally, can you sort the CSS properties? I thought we had a lint check for that, not sure why it passed.

**Comment by johngruen** [comment]
@shane-tomlinson i have a feeling at least some of the issues you found were already there, but i can run all of these down for sure.

Thanks for the review!

**Comment by johngruen** [comment]
@shane-tomlinson i resolved all of your issues and repushed!

**Comment by shane-tomlinson** [comment]
Doing some more testing:

### Permission label is not aligned with the value above it

Screenshot 2019-04-25 at 11 48 43

### Really long addresses appear under the "Make primary" button

Screenshot 2019-04-25 at 11 51 04

### Make primary text doesn't fit the button on mobile

Screenshot 2019-04-25 at 11 52 30

### Seems like a "tablet" breakpoint has gone missing (or a result of the long email/display name?)

Screenshot 2019-04-25 at 11 53 25

### Really long email does not fit in the re-enter password modal for "recovery key"

Screenshot 2019-04-25 at 11 54 38

### Change password does not fit on the screen in some resolutions and causes the TOS/PP links to be too far to the right

Screenshot 2019-04-25 at 12 02 30

### There is a similar problem when signing up, the entire password strength bubble is not within the viewport

Screenshot 2019-04-25 at 12 06 29

**Comment by shane-tomlinson** [comment]
@johngruen - looks like there are a bunch of conflicts, I'm guessing with the PR that removed the `trustedUI` breakpoint.

**Comment by johngruen** [comment]
@shane-tomlinson okay, i think we're good...those password balloons are annoying

**Comment by shane-tomlinson** [comment]
@johngruen found one more really long email issue - on the permissions screen:

### Desktop

Screenshot 2019-04-29 at 16 26 05

### Mobile

Screenshot 2019-04-29 at 16 24 56

With that fixed, I'm comfortable with an r+

**Comment by johngruen** [comment]
@shane-tomlinson fixed ur requestst

**Pull request merged**